jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -27,7 +27,7 @@
       :items="dataset_items"
       :selected.sync="dataset_selected"
       :multiselect="multiselect"
-      :show_if_single_entry="false"
+      :show_if_single_entry="true"

```

Comment:
This needs to be "true" to fix a buggy GUI behavior I saw. Also this is now consistent with subset drop-down behavior.

Reply:
what buggy behavior?  Most places we do have the data dropdown hide if there is only one data entry.  Its not a huge deal to change this, but I'd rather try to fix the underlying bug.

Reply:
I don't quite remember. It was late at night. Might be something about loading/unloading things from viewer vs plugin updates. I think this change should go in regardless.

Reply:
Are you proposing we should change this behavior app-wide then?  The whole idea is that when there is only a single image, its obvious what is being used as input and this can save space and make the workflow seem simpler.  Unless we change this app-wide, please do try to reproduce and file a report for the bug so that we can fix it.

Reply:
When I am doing photometry, I like to see what the image is on the plugin regardless. Is it just me?

Anyways, the multi-select isn't broken on Plot Options in the same session but broken for this plugin. I wonder why. Will investigate.

Reply:
Hmm, despite this change, it just disappeared again. I see this in dev console. What does it mean?

<img width="395" alt="Screenshot 2024-01-29 120837" src="https://github.com/spacetelescope/jdaviz/assets/2090236/90a64b32-e43a-460e-a97b-bdac54d0825f">

Reply:
What is this doing and why is it triggering? How do I fix it for Cubeviz?

https://github.com/spacetelescope/jdaviz/blob/c5b0c984311f9c0a38d01fff82ca907d27685d64/jdaviz/components/plugin_dataset_select.vue#L23

https://github.com/spacetelescope/jdaviz/blob/c5b0c984311f9c0a38d01fff82ca907d27685d64/jdaviz/components/plugin_dataset_select.vue#L64-L74

Reply:
I think Kyle's suggestion to remove the offending code completely fixed this problem. A quick check showed me both Cubeviz and Imviz worked fine with multi-select now.